pages/1.13/security/ent/tls-ssl/exhibitor/index.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: layout.pug
+navigationTitle:  Securing Exhibitor with mutual TLS
+title: Securing Exhibitor with mutual TLS
+menuWeight: 500
+excerpt: Securing DC/OS with a TLS enabled Exhibitor ensemble
+enterprise: true
+---
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+By default, the Exhibitor HTTP service is open to any client that can reach port 8181 on a master node. This documentation chapter describes a method for protecting the Exhibitor service from unauthorized access. Once enabled, HTTP clients must access Exhibitor through Admin Router; thus applying the Admin Router access control policy to the exhibitor service.
+
+*Note: When accessing Exhibitor through Admin Router (**https://master_host/exhibitor**), authenticated users must have the `dcos:adminrouter:ops:exhibitor` privilege*
```

Comment:
:rocket: 

I think this is really clear. Cool.

Reply:
Please use the proper format for notes. You can find the directions here in the style guide:

https://wiki.mesosphere.com/display/DOCS/Documentation+Style+Guide#DocumentationStyleGuide-Notes